### PR TITLE
fix(cloud-templates): allow adding properties without default value

### PR DESCRIPTION
**Pull request by marstamm**
related to https://github.com/camunda/camunda-modeler/issues/3140

### CODE REVIEW

File: src/provider/cloud-element-templates/util/propertyUtil.js [hunk 1]
```diff
@@ -168,8 +168,11 @@ export function setPropertyValue(bpmnFactory, commandStack, element, property, v
     property
   };
 
+  let propertyHandled = false;
```
Comment (nikku) [inline src/provider/cloud-element-templates/util/propertyUtil.js:171]: What if we early return if handled instead? This pattern looks odd to me.

File: src/provider/cloud-element-templates/util/propertyUtil.js [hunk 2]
```diff
@@ -188,6 +191,7 @@ export function setPropertyValue(bpmnFactory, commandStack, element, property, v
 
   // property
   if (type === 'property') {
```
Comment (nikku) [inline src/provider/cloud-element-templates/util/propertyUtil.js:193]: Potential improvement: Get rid of magic value.

File: src/provider/cloud-element-templates/util/propertyUtil.js (hunk 1 above)
Comment (marstamm) [reply src/provider/cloud-element-templates/util/propertyUtil.js:171]: We can't return early, same as we can't switch to `if else` branches. One property might have multiple handlers within the file, which can add more commands to the `commands` array.

We could return early for a simple `property` value, as this only has one handler, but this pattern would guarantee that it works for all future properties.

Comment (nikku) [reply src/provider/cloud-element-templates/util/propertyUtil.js:171]: Any other idea how we can re-structure this code path?

Comment (marstamm) [reply src/provider/cloud-element-templates/util/propertyUtil.js:171]: My Proposal:
- Turn the paths into handler functions
- map property name to list of handlers
- if handler list is empty, throw error
- else: execute the commands, if any

I will try have this ready for the HoC tomorrow. Moving to "in progress" until then

File: src/provider/cloud-element-templates/util/propertyUtil.js (hunk 2 above)
Comment (marstamm) [reply src/provider/cloud-element-templates/util/propertyUtil.js]: done in [13c19df](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/763/commits/13c19df796f42a585f7204fea25678b75ae70f9c)

File: src/provider/cloud-element-templates/util/propertyUtil.js (hunk 1 above)
Comment (marstamm) [reply src/provider/cloud-element-templates/util/propertyUtil.js]: We decided to go with the NoOp solution to fix this bug, and to get rid of the `multi-command-executor` in the future, potentially during spring cleaning

**Pull request merged**